proto/umee/leverage/v1/leverage.proto

```diff
@@ -117,4 +117,22 @@ message Token {
   // list of allowed tokens.
   string symbol_denom = 10 [(gogoproto.moretags) = "yaml:\"symbol_denom\""];
   uint32 exponent     = 11 [(gogoproto.moretags) = "yaml:\"exponent\""];
+  
+  // Allows lending of this token. Note that withdrawing is always enabled.
+  // Disabling lending would be one step in phasing out an asset type.
+  bool enable_lend = 12;
+
+  // Allows borrowing of this token. Note that repaying is always enabled.
+  // Disabling borrowing would be one step in phasing out an asset type, but
+  // could also be used from the start for asset types meant to be collateral
+  // only, like meTokens.
+  bool enable_borrow = 13;
+
+  // This should only be used to eliminate an asset completely. A blacklisted
+  // asset is treated as though its oracle price is zero, and thus ignored by
+  // calculations such as collateral value and borrow limit. Can still be repaid
+  // or withdrawn, but not liquidated. A blacklisted token must have enable_lend
+  // and enable_borrow set to false. Such tokens can be safely removed from the
+  // by the oracle and price feeder as well.
+  bool blacklist = 14;
```

Comment:
Runs the risk of being interpreted as just a combination of the other two being disabled.

This is more serious in that it also forces the value of the coin to zero in all calculations, which would have all kinds of consequences - we might look for a name that communicates that, but `blacklist` at least shows the severity.

Reply:
In https://github.com/umee-network/umee/pull/913#discussion_r875948663 I was thinking to remove this field. 

What it would take to unblackist an asset?

Reply:
Technically, only a governance proposal with the `blacklisted` field set back to false.

In reality, it seems unlikely that an accepted asset could become so nasty we have to blacklist it, but then recover to a point where we need to reverse the decision.